pytensor/scan/op.py

```diff
@@ -1134,60 +1173,60 @@ def make_node(self, *inputs):
                 )
 
         argoffset += len(self.outer_sitsot(inputs))
-        # Check that the shared variable and their update rule have the same
+        # Check that the untraced (u) sit-sot variable and their update rule have the same
```

Comment:
What does (u) mean

Reply:
It's because I use the letter u in the variable names below. So am abbreviation